### PR TITLE
remove javadoc comments from representation #17

### PR DESCRIPTION
**Pull request by richarda23**
Hi, 
- added a handler to remove Javadoc comments
- this was exposed by alphametics solution that contained JavaDoc, so removed those from the representation.

### CODE REVIEW

File: src/test/resources/exercises/alphametics/representation [hunk 1]
```diff
@@ -19,9 +19,6 @@ public class PLACEHOLDER_1
         return PLACEHOLDER_11.get().orElseThrow(UnsolvablePuzzleException::new);
     }
 
-    /**
-     * Returns the list of distinct characters in this alphametic puzzle.
-     */
```
Comment (iHiD) [inline src/test/resources/exercises/alphametics/representation:24]: What's the reason for removing these comments? They seem useful to me :)

File: src/main/java/representer/normalizer/CommentNormalizer.java [hunk 2]
```diff
@@ -22,4 +24,12 @@ public Node visit(LineComment n, String arg) {
...
         return null;
     }
+
+    /**
+     * Remove nodes of type JavadocComment
+     */
+    @Override
+    public Node visit(JavadocComment n, String arg) {
+        return null;
+    }
```
Comment (iHiD) [inline src/main/java/representer/normalizer/CommentNormalizer.java:34]: Nice. Thanks :)

File: src/test/resources/exercises/alphametics/representation (hunk 1 above)
Comment (richarda23) [reply src/test/resources/exercises/alphametics/representation:24]: To make the tests pass. If we remove the Javadoc comments, then this current representation text does not match what is generated by after parsing the submitted solution, . As I understand it this issue  was to remove Javadoc comments?

**Comment by mirkoperillo** [comment]
@richarda23 thank you for your contribution I will review it ASAP

File: src/test/resources/exercises/alphametics/representation (hunk 1 above)
Comment (ErikSchierboom) [reply src/test/resources/exercises/alphametics/representation:24]: It was. The comments should indeed be removed from that file 👍

**Review by ErikSchierboom — APPROVED** [review]
Great improvement! This will help a lot.

File: src/test/resources/exercises/alphametics/representation (hunk 1 above)
Comment (iHiD) [reply src/test/resources/exercises/alphametics/representation:24]: Ah. I missed which file this was in. Sorry. Thanks!

**Review by mirkoperillo — APPROVED** [review]
@richarda23 good work, thank you

**Pull request merged**